packages/react/src/components/AppShell/app-shell.scss

```diff
@@ -0,0 +1,49 @@
+/**
+ * Copyright (c) 2023, WSO2 LLC. (https://www.wso2.com). All Rights Reserved.
+ *
+ * WSO2 LLC. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+.oxygen-app-shell {
+  width: 100%;
+  display: flex;
+  overflow: hidden;
+  flex-grow: 1;
+  flex-direction: column;
+
+  .oxygen-header {
+    z-index: 1201;
+  }
+
+  .oxygen-app-shell-content {
+    display: flex;
+    overflow: auto;
+    flex-grow: 1;
+    transition: all 0.5s;
+
+    .oxygen-app-shell-main-wrapper {
+      display: flex;
+      overflow: auto;
+      flex-grow: 1;
+      flex-direction: column;
+
+      .oxygen-app-shell-main {
+        flex-grow: 1;
+        padding: 24px;
```

Comment:
WDYT about using the spacing var? https://mui.com/material-ui/customization/spacing/#custom-spacing
We will have to implement this in other components as well if we decide to go ahead.

Reply:
+1

Created a new variable and used the `spacing(3)` from the theme since it was not directly exposed as a var.

Fixed with https://github.com/wso2/oxygen-ui/pull/29/commits/0b2fc6d33258d6c0668ef6ffd0080d950d45014d